config/samples/mattermost_v1beta1_mattermost.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: mattermost.mattermost.com/v1beta1
+kind: Mattermost
+metadata:
+  name: mattermost-sample
+spec:
+  # Add fields here
+  foo: bar
```

Comment:
this is a valid field?

Reply:
That's just a sample config that was auto-generated. As I mentioned in the original PR description, more work is needed to finalize the auto-generated files that will come later.

Reply:
sgtm, but we can remove this to not have these invalid fields in the repo, but it is up to you